app/domain/authentication/authn_k8s/validate_pod_request.rb

```diff
@@ -74,8 +75,8 @@ def validate_resource_restrictions
       def validate_container
         unless container
           raise Errors::Authentication::AuthnK8s::ContainerNotFound,
-            container_name,
-            k8s_host.conjur_host_id
+                container_name,
```

Comment:
seems like a large tab no?

Reply:
code climate asked for them to be aligned with `Errrors`